specification/gedcom-3-structures-4-enumerations.md

```diff
@@ -230,14 +230,83 @@ and applications should be prepared to encounter non-current values.
+| `OTHER` | A value not listed here; should have a `PHRASE` substructure. |
+
+It is expected that many name forms will have no `TYPE`.
+The researcher-preferred name form is indicated by its being the first `FORM` of the `NAME`, not by any `TYPE` value.
+
+### `g8:enumset-NAME-PART-TYPE`
+
+| Value | Meaning                       |
+| ----- | :---------------------------- |
+| `ADOPTED` | Given as part of being adopted into a family. |
+| `DIVORCED` | Name used after a divorce. |
+| `ESTATE` | House name, farm name, or name after moving into or marrying into a house/farm. Implies `LOCATION`. Incompatible with `SURN`. | |
+| `FORMAL` | A name only used official, formal settings. |
+| `GENERAL` | A name used in a wide variety of settings, both formal and informal. |
+| `GENERATIONAL` | A name part shared by particular generation of a family (i.e. siblings or first cousins, but not their parents or children). Implies a cultural pattern of sharing this part, not just a particular family's aesthetic naming patterns. |
+| `GIVN` | A name given to an individual by someone's choice, rather than dictated by the rules of the culture, often to be used to identify that individual that individual and differentiate them from other members of the same family or community. Incompatible with `SURN`. |
+| `HONORIFIC` | A word or phrase attached to a name in formal or polite context to indicate station, such as "Miss", "Doctor", "さん", "様", "mademoiselle", and so on. |
+| `IMMIGRANT` | Name assumed at the time of immigration. |
+| `INFORMAL` | A name only used in casual, intimate, or informal settings. |
+| `LEGAL` | A name used for legal and official documents, but not in daily use. |
+| `LOCATION` | A name indicating a location of note, such as a city associated with the person. Often includes "of" or "from" type particles. Incompatible with `SURN`. |
+| `MAIDEN` | Maiden name, name before first marriage. |
+| `MARRIED` | Married name, assumed as part of marriage. |
+| `MATERNAL` | A name inherited from the individuals' mother's family. Implies `SURN`. |
+| `MATRONYMIC` | A name of the individual's mother, possibly with a matronymic modifier. |
+| `NICK` | A descriptive or familiar name that is used instead of, or in addition to, one’s official or legal name. Some cultures use this for any name that is not used in legal documents, others only for names that would be inappropriate in formal settings. |
+| `NPFX` | Text that appears on a name line before the given and surname parts of a name. Implies that the person attaches this part to their name, but does not consider it part of the name itself. |
+| `NSFX` | Text which appears on a name line after or behind the given and surname parts of a name. Implies that the person attaches this part to their name, but does not consider it part of the name itself. |
+| `PARTICLE` | A name part that connects or modifies other name parts but is not itself considered a name, like "of" or "son of". | 
```

Comment:
Discussion: what about `OCCUPATION`.  That would seem to be a common name part in some cultures and times in history.